presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -154,7 +154,7 @@
     private FunctionAssertions functionAssertions;
     private List<ListenableFuture<?>> futures;
 
-    @BeforeSuite
```

Comment:
It looks like the commit message is about something different.

Reply:
No it's not. `@AfterSuite` happens too late (when all tests finish), so it's as there was no tear down method at all.